src/components/DataTable/README.md

```diff
@@ -334,7 +334,7 @@ function FullDataTableExample() {
             'numeric',
           ]}
           headings={[
-            'Product',
+            <TextStyle variation="strong">Product</TextStyle>,
```

Comment:
I'm not sure I would update this example. Sort of implies it's a pattern we want to promote.

```suggestion
            'Product',
```

Reply:
Okay, fair enough